cmd/rpcdaemon/commands/trace_api_adhoc.go

```diff
@@ -2,34 +2,48 @@ package commands
 
 import (
 	"context"
+	"fmt"
+
+	"github.com/ledgerwatch/turbo-geth/common"
+	"github.com/ledgerwatch/turbo-geth/core/types"
+	"github.com/ledgerwatch/turbo-geth/rpc"
 )
 
+// CallParam a parameter for a eth_callMany routine
+type CallParam struct {
+	tx         types.Transaction
+	traceTypes []string
+}
+
+// CallParams array of callMany structs
+type CallParams []CallParam
+
 // Call Implements trace_call
-func (api *TraceAPIImpl) Call(ctx context.Context, req TraceFilterRequest) ([]interface{}, error) {
+func (api *TraceAPIImpl) Call(ctx context.Context, call CallParam, blockNr rpc.BlockNumber) ([]interface{}, error) {
 	var stub []interface{}
-	return stub, nil
+	return stub, fmt.Errorf("function eth_call not implemented")
```

Comment:
I think they should say `trace_call not implemented` and so on, not `eth_call`, because `eth_call` is implemented :)

Reply:
Updated. Sorry about that. I'll try to be more careful in the future. It's late in the week...